### PR TITLE
Adding jenkins

### PR DESCRIPTION
**Pull request by genievot**
Adding Jenkins to default Okteto app catalog

The icon needs to be adjusted.
Please let me know if anything to adjust.

### CODE REVIEW

File: jenkins/values-okteto.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+jenkins:
+  master:
+    adminUser: "admin"
+    # adminPassword find with: printf $(kubectl get secret --namespace contribution-genievot jenkins -o jsonpath="{.data.jenkins-admin-password}" | base64 --decode);echo
```
Comment (rberrelleza) [inline jenkins/values-okteto.yaml:4]: Is the password always generated, or can the user provide it?

File: jenkins/values-okteto.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    existingSecret: {}
+    env: []
+    resources:
+      requests:
+        memory: 1Gi
+        cpu: 1
+      limits:
+        memory: 1Gi
+        cpu: 1
+    destination: "s3://jenkins-data/backup"
```
Comment (rberrelleza) [inline jenkins/values-okteto.yaml:26]: I'd leave this empty, since this bucket will not work for anyone.

File: jenkins/values-okteto.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+      tag: "0.2.0"
+    extraArgs: []
+    existingSecret: {}
+    env: []
+    resources:
+      requests:
+        memory: 1Gi
+        cpu: 1
+      limits:
+        memory: 1Gi
```
Comment (pchico83) [inline jenkins/values-okteto.yaml:24]: How many cpu/memory jenkins consumes on average for small CI jobs?

File: jenkins/templates/NOTES.txt [hunk 4]
```diff
@@ -0,0 +1,5 @@
+Visit Url visible in okteto cloud or use okteto mobile app from playstore to get url.
+######
+Get Password with: 
+printf $(kubectl get secret --namespace contribution-genievot jenkins -o jsonpath="{.data.jenkins-admin-password}" | base64 --decode);echo
```
[the remark this answers is not in the document]
Comment (genievot) [reply jenkins/templates/NOTES.txt:4]: ah better, didn't notice.

File: jenkins/values-okteto.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (genievot) [reply jenkins/values-okteto.yaml:4]: always random, default jenkins from helm stable repo have those rules!

File: jenkins/values-okteto.yaml (hunk 2 above)
Comment (genievot) [reply jenkins/values-okteto.yaml:26]: ok

File: jenkins/values-okteto.yaml (hunk 1 above)
Comment (genievot) [reply jenkins/values-okteto.yaml:4]: always random, default jenkins from helm stable repo have those rules!

Comment (genievot) [reply jenkins/values-okteto.yaml:4]: it has an endpoint to provide repo links etc. I am overriding those settings in parent's values.

File: jenkins/values-okteto.yaml (hunk 3 above)
Comment (genievot) [reply jenkins/values-okteto.yaml:24]: I found some information regarding this at Jenkins official docs https://jenkins.io/doc/book/hardware-recommendations/
I think resources can be inside values-okteto.yaml so users can change them according to their needs...

**Comment by pchico83** [comment]
@genievot could you also add a readme with an intro and a link to a tutorial sample?

**Comment by genievot** [comment]
Sure, i will do it soon

**Comment by pchico83** [comment]
@genievot I am going to close this one. We want to focus Okteto Cloud on the pre-commit dev flow, and jenkins is oriented on CI tasks after commit.

**Comment by genievot** [comment]
sure, thanks

**Pull request closed without merging**